source/appModules/code.py

```diff
@@ -7,41 +7,23 @@
 
 import appModuleHandler
 import controlTypes
-from NVDAObjects.IAccessible.ia2Web import Editor
 from NVDAObjects.IAccessible.chromium import Document
-from NVDAObjects.IAccessible.ia2TextMozilla import MozillaCompoundTextInfo
 from NVDAObjects import NVDAObject, NVDAObjectTextInfo
 
 
-class VSCodeEditorTextInfo(MozillaCompoundTextInfo):
-	def _isCaretAtEndOfLine(self, caretObj) -> bool:
-		# #17039: IAccessibleText::textAtOffset with IA2_OFFSET_CARET is way too costly in VSCode.
-		# And doesn't actually work correctly in Chromium yet anyway.
-		# So it is best not to try detecting for now.
-		return False
-
-
-class VSCodeEditor(Editor):
-	TextInfo = VSCodeEditorTextInfo
```

Comment:
This seems to be an API breaking change

Reply:
No, the removed code was only added in pr #17051.